### ISSUE TITLE
Create page to show third party lantern users that they should download real lantern.

### ISSUE THREAD

**Issue by Derekf5** [getlantern/lantern#4687]
@p3rcya could you describe this for everyone here.

**Comment by p3rcya** [comment]
Our android users are predominately using older version without feed. So the only way to target them is to redirect their traffic. We have two options:
1) Redirect their http traffic probabilistically and inform users that they have to upgrade (provide apk download address) to continue to use Lantern. We will offer a bypass button. 

2) Replacing all images and ads with the same message as in 1)

Please note: This should be a call for upgrade rather than an ad for Pro. We don't want users to think that we are hijacking their traffic to promote Pro. Rather, we are only hijacking traffic to ask them to upgrade.

**Comment by Derekf5** [comment]
@p3rcya is this only for mobile?

**Comment by p3rcya** [comment]
@Derekf5 desktop+mobile

**Comment by Derekf5** [comment]
Hey All here are the specs for the desktop page.  It closely resembles our webpage so some of the code can be reused (the header, font and button styles etc.)  @fffw mentioning this to you since you seem to be the person who will be putting up the site.

<img width="685" alt="screen shot 2016-07-15 at 12 56 17 pm" src="https://cloud.githubusercontent.com/assets/7864729/16882178/94d0d71e-4a8b-11e6-9789-43108710a1dd.png">
<img width="684" alt="screen shot 2016-07-15 at 12 56 24 pm" src="https://cloud.githubusercontent.com/assets/7864729/16882179/94d4aac4-4a8b-11e6-8077-b240ef56a22c.png">

**Comment by Derekf5** [comment]
Here is the mobile version. CC @atavism @fffw 
<img width="240" alt="screen shot 2016-07-15 at 12 58 27 pm" src="https://cloud.githubusercontent.com/assets/7864729/16882225/da896ec4-4a8b-11e6-8a07-2bdbb3794e4b.png">
<img width="239" alt="screen shot 2016-07-15 at 12 58 21 pm" src="https://cloud.githubusercontent.com/assets/7864729/16882230/dff3c3c8-4a8b-11e6-9a39-113411ecd2b1.png">

**Comment by Derekf5** [comment]
Here is the mobile version. CC @atavism @fffw 
<img width="297" alt="screen shot 2016-07-15 at 1 06 46 pm" src="https://cloud.githubusercontent.com/assets/7864729/16882467/2669d094-4a8d-11e6-9658-88672f1cb1b6.png">
<img width="297" alt="screen shot 2016-07-15 at 1 06 52 pm" src="https://cloud.githubusercontent.com/assets/7864729/16882468/267923fa-4a8d-11e6-88e9-23d1be246d69.png">

**Comment by Derekf5** [comment]
@myleshorton @joesis   just thinking that we should start working on this at some point so that we can turn it on once auto updates reach 100%

**Comment by p3rcya** [comment]
Edited the title to reflect that it's not limited to third party client, but also older 2.x users or even some 3.x with really out dated versions

**Comment by joesis** [comment]
It sems unnecessary for authentic desktop clients as they will auto-update to newest version.

**Comment by p3rcya** [comment]
@joesis I don't think we can differentiate 2.x clients with third party client anyway at serverend

**Comment by p3rcya** [comment]
This might not be necessary. Let's see the versions breakdown after 100%

**Comment by p3rcya** [comment]
cc @Derekf5 @myleshorton @GLantern @WolfLantern We have 65% of users stuck with a version without payment wall. So this should be a top priority.

**Comment by Derekf5** [comment]
yes @p3rcya it will likely make it into next sprint.

**Comment by carsonefv** [comment]
Here is the updated design @p3rcya @joesis @Derekf5 
(note: please insert the end-date when implementing)

<img width="822" alt="screenshot 2017-03-01 15 44 40" src="https://cloud.githubusercontent.com/assets/4400763/23480513/2fa6ef62-fe96-11e6-9f4b-ca902cd1ca9b.png">

**Comment by joesis** [comment]
The page is alive at https://getlantern.org/outdated.html with a few differences need to resolve:

1. The logo has no shadow. I was using the logo find from the desktop UI repo, which is without shadow. @carsonefv Can you send me an image with shadow applied?

2. As we don't have an end date estimated currently, and the user may not really care about the absolute date, so I rephrased the text as "You are using an old version of Lantern which will soon no longer work." atm.

3. A `font-weight` of 100 and 300 looks identical in English. Maybe we differentiate them by the color? They do look different on firefox in Chinese, see below.

![image](https://cloud.githubusercontent.com/assets/22091642/23849152/60ff66c4-0815-11e7-91d2-aaba1a969c69.png)

**Comment by carsonefv** [comment]
Thanks @joesis !
- Here is a link to the logo with shadow: https://www.dropbox.com/s/qh0beo3fkk55d0s/LanternOfficial2016%20square%26circle.svg?dl=0
- the text change sounds good to me
- Can you try specifying the font weights as "thin" for 100 and "light" for 300, instead of using the numbers? If that doesn't work, just leave as is, not a big deal if they are the same weight

**Comment by joesis** [comment]
Thanks, @carsonefv. The logo with shadow is applied. `thin` and `light` is unrecognizable by Chrome, and maybe the CSS standard too (https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight), so I left it as is.
